modelexpress_client/python/modelexpress/weight_transfer/transport/nccl_m2n_bootstrap.py

```
@@ -0,0 +1,256 @@
+def _abort_with_note(m2n: _M2nBinding, comm: int, error: BaseException) -> None:
+    """Abort locally without replacing the original failure.
+
+    ncclCommAbort is synchronous and may itself hang. A process supervisor must
+    enforce the hard whole-attempt termination deadline.
+    """
+
+    if not comm:
+        return
+    try:
+        m2n.comm_abort(comm)
+    except BaseException as abort_error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)